proto/redpanda/core/admin/v2/shadow_link.proto

```diff
@@ -193,24 +193,27 @@ message ShadowLinkClientOptions {
     optional TLSSettings tls_settings = 4;
     // Authentication settings
     optional AuthenticationConfiguration authentication_configuration = 5;
-    // Max metadata age
+    // Max metadata age.
```

Comment:
FYI to make this look good in docs I think we want an extra newline.

https://redpandadata.atlassian.net/wiki/spaces/CORE/pages/1360068616/Admin+API+v2#Add-a-comment-with-the-rpc-definition-to-create-an-operation-summary-and-description